tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1174,26 +1174,37 @@ def test_backup_uss_file(ansible_zos_module, backup):
     src = "/etc/profile"
     dest = "/tmp/profile"
     backup_name = None
+    if backup:
```

Comment:
We would want to double check this change here after the original logic is reverted back meaning a user can specify backup:true , or backup:true and backup_name:foo